### PR TITLE
Support Falcon request validation

### PR DESCRIPTION
**Pull request by pythonishvili**
Hello, I am looking for a way to validate [Falcon](https://falconframework.org/) Request against Swagger schema. If you consider it interesting, I am ready to continue development as I have a couple of ideas to investigate.

### CODE REVIEW

File: flex/http.py [hunk 1]
```diff
@@ -15,6 +16,12 @@
 except ImportError:
     _tornado_available = False
 
+_falcon_available = True
```
Comment (pipermerriam) [inline flex/http.py]: Can you put this in an `else` clause for the `try/except` below.

**Comment by pipermerriam** [comment]
This seems completely reasonable to me.

File: flex/http.py (hunk 1 above)
Comment (pythonishvili) [inline flex/http.py]: Sure, this was my original intention but I wanted to be consistent with `_tornado_available`. I can change both in a separate commit. What do you think?

Comment (pythonishvili) [inline flex/http.py]: OK, it's done.

Comment (pipermerriam) [inline flex/http.py]: Thanks for this.  I didn't realize the above tornado stuff was done the same way.  Appreciate you cleaning it up as well.

**Comment by pipermerriam** [comment]
This is now released on pypi as `5.9.0`  enjoy :)

**Comment by pythonishvili** [comment]
thx!

**Pull request merged**